web/src/pages/export.tsx

```diff
@@ -0,0 +1,60 @@
+import { CircularIndicator } from "@/components/CircularIndicator";
+import { Roadmap } from "@/components/dashboard/Roadmap";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
+import { TaskBody } from "@/components/TaskBody";
+import { useRoadmap } from "@/lib/data-hooks/useRoadmap";
+import { ROUTES } from "@/lib/domain-logic/routes";
+import { exportComponentsAsPDF } from "@/lib/roadmap/exportRoadmapAsPdf";
+import { useRouter } from "next/router";
+import { ReactElement, useEffect, useState } from "react";
+
+const ExportPage = (): ReactElement => {
```

Comment:
is this component being used / linked to anywhere?

Reply:
This page is not linked anywhere in the application. Dillon mentioned that this would likely be under a future flag for the next several months, and will go through several iterations before we figure out the final state of the PDF. We decided to render the functionality on a separate page, as opposed to adding it to an existing page, such as the profile page.